entity-service/src/main/kotlin/com/egm/stellio/entity/repository/Neo4jRepository.kt

```diff
@@ -393,15 +416,22 @@ class Neo4jRepository(
         return session.query(query, mapOf("entitiesIds" to entitiesIds), true).map { it["id"] as String }
     }
 
-    fun getPropertyOfSubject(subjectId: String, propertyName: String): Property {
-        val query = """
+    fun getPropertyOfSubject(subjectId: String, propertyName: String, datasetId: URI? = null): Property? {

```

Comment:
I'm not really happy with returned Property being nullable, it forces to make (always dangerous) not null assertions

Reply:
I will fix this

Reply:
In the previous method, datasetId parameter is a string. In this one, it is an URI. Better be consistent.